### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by DTuniverse**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
-</head>
-<body>
-    
-</body>
+  </head>
+  <body>
+    <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (DaniilPlayGround) [inline index.html:10]: Could you remove the 'Enjoy your stay' string and make the 'Welcome to Potionfy!' string red?

Comment (DTuniverse) [reply index.html:10]: Sure can do!

**Review by DaniilPlayGround — CHANGES_REQUESTED** [review]
Please change the code as per my comments on the review.

**Review by DaniilPlayGround — APPROVED** [review]
Well done Harry!

**Pull request merged**